### PR TITLE
Fix the indexing problem in typedarray_prototype_index_of & last_index_of.

### PR DESCRIPTION
**Pull request by orkvi**
JerryScript-DCO-1.0-Signed-off-by: Virag Orkenyi orkvi@inf.u-szeged.hu

### CODE REVIEW

File: jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c [hunk 1]
```diff
@@ -1373,42 +1366,42 @@ ecma_builtin_typedarray_prototype_last_index_of (ecma_typedarray_info_t *info_p,
   {
     from_index = info_p->length - 1;
   }
+
```
Comment (rerobika) [inline jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c:1369]: Do not remove this line.

File: jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c [hunk 2]
```diff
@@ -1360,6 +1352,7 @@ ecma_builtin_typedarray_prototype_last_index_of (ecma_typedarray_info_t *info_p,
 #endif /* ENABLED (JERRY_BUILTIN_BIGINT) */
 
   uint32_t from_index;
+  //uint32_t limit = info_p->length * info_p->element_size;
```
Comment (rerobika) [inline jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c:1355]: Remove this comment.

File: jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c [hunk 3]
```diff
@@ -1373,42 +1366,42 @@ ecma_builtin_typedarray_prototype_last_index_of (ecma_typedarray_info_t *info_p,
...
-    ecma_number_t num_var;
-    if (ECMA_IS_VALUE_ERROR (ecma_op_to_integer (args[1], &num_var)))
+    if (ECMA_IS_VALUE_ERROR (ecma_builtin_helper_uint32_index_normalize (args[1],
+                                                                         info_p->length,
+                                                                         &from_index)))
     {
       return ECMA_VALUE_ERROR;
     }
+    ecma_number_t to_int;
+    ecma_op_to_integer (args[1], &to_int);
```
Comment (rerobika) [inline jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c:1379]: This operation can throw an error. Please handle it.

**Review by rerobika — CHANGES_REQUESTED** [review]

**Review by rerobika — APPROVED** [review]
LGTM

**Review by dbatyai — APPROVED** [review]
LGTM

**Pull request merged**